libraries/provision/install_sync_gateway.py

```diff
@@ -132,6 +135,15 @@ def install_sync_gateway(cluster_config, sync_gateway_config, sg_ce=False):
         "xattrs": ""
     }
 
+    if is_cbs_ssl_enabled(cluster_config) and get_sg_version(cluster_config) >= "1.5.0":
```

Comment:
This is a string comparison. compare_versions(version_one, version_two): in keywords.utils might be a better fit.